recipes-kernel/linux/linux-hikey_4.5.bb

```diff
@@ -0,0 +1,18 @@
+require linux-hikey.inc
+
+DESCRIPTION = "Hisilicon 4.5 Kernel"
+
+PV = "4.5+git${SRCPV}"
+
+SRCREV_kernel="623d1d7bbd697e5793a135d5334e02c0e9889ba1"
+
+SRC_URI = "git://github.com/xin3liang/linux;branch=hikey-tracking-integration-devel-drm;name=kernel \
+           file://defconfig  \
```

Comment:
While this kernel is targeted for hikey, we might want to rename the recipe as this is actually coming from a personal repository, and not really generic.

Something like linux-hikey-devel-drm-4.5 might be better.